tailwind.config.ts

```diff
@@ -22,8 +22,11 @@ const config: Config = {
           "linear-gradient(90deg, #a3bac3 50%, #f6a200 50%, #f6a200 100%)",
       },
       colors: {
-        primary: "#002B38",
-        secondary: "#F6A200",
+        // primary: "#002B38",
```

Comment:
I think you should just delete those comments, in case of needed to go back you can revert that change in the future